### ISSUE TITLE
A new patch has been released

### ISSUE THREAD

**Issue by Amattia** [Pyroblasty/pyroblasty.github.io#2]
Hi, i have a question for you, currently new have been implemented with the newest patch, I would like to know if the project will be updated, or alternatively to know if I could get a few tips to refresh it...

I have some trouble understanding about the operation of the code generation (in the url).

Best regards

**Comment by Pyroblasty** [comment]
Yeah, this project needs some major update. I've promised to fix it a month
ago already, unfortunately I've never had the time to actually do it. As
soon as I've added the Assassin mastery in my eprsonal build, Expose
Weakness was already out on the PBE.

Hopefully I'll get to do it in the next few days. Keep reminding this to
me! :D

2016-03-07 17:45 GMT+01:00 Anedda Mattia notifications@github.com:

**Comment by Amattia** [comment]
Hi, news ? :)
